### DISCUSSION TITLE
How to upgrade specifically OnedriveGUI ?

### DISCUSSION THREAD

**Discussion by moijeux1** [bpozdena/OneDriveGUI#197] [Q&A]
Hi,
Since some day, I have tried to install onedrive and onedrive client. I try more time with the old version and the new version. But i had have problems each time : impossible to synchronize, no action with the client, bad folder to synchronize the files, new version client doesn't see Onedrive 2.5.2 etc.
 System:
  Kernel: 6.8.0-45-generic arch: x86_64 bits: 64 compiler: gcc v: 13.2.0 clocksource: tsc
  Desktop: Cinnamon v: 6.2.9 tk: GTK v: 3.24.41 wm: Muffin v: 6.2.0 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22 Wilma base: Ubuntu 24.04 noble

I  don't know what information you need.

I decide to restart the installation of Mint and the old Onedrive and Onedrive Client. But the system notifies me the new version is available... How do I upgrade Specifically Onedrive and OnedriveGUI client knowing that we can't do it through sudo apt update?
Thanks
Dom

**Comment by bpozdena** [comment]
Please create a new issue and provide all requested details from the issue template.
